sarxarray/stack.py

```diff
@@ -109,6 +110,85 @@ def _amp_disp(self, chunk_azimuth=500, chunk_range=500):
 
         return amplitude_dispersion
 
+    def multi_look(self, window_size, method="coarsen", statistics="mean", chunk=1000, compute=True):
+        """
+        Perform multi-looking on a Stack, and return a Stack.
+
+        Parameters
+        ----------
+        window_size : tuple
+            Window size for multi-looking, in the format of (azimuth, range)
+        method : str, optional
+            Method of multi-looking, by default "coarsen"
+        statistics : str, optional
+            Statistics method for multi-looking, by default "mean"
+        chunk : int, optional
```

Comment:
Can we by default keep the original chunksize(if exists) in `self._obj`

Reply:
After discussion we agreed it does not make sense to use the original chunksize, since the resolution already changed. Instead we can estimate the new chunksize based on the multi-look window size and the original chunk size.

Reply:
right, now the code uses existing chunk size and reduces it based on window size, see my [last commit ](https://github.com/MotionbyLearning/sarxarray/pull/32/commits/6f84bb76552b9a7cf87c97789c4e99ff0e168ba1).